docs/resources/workspace_desktop.md

```diff
@@ -122,6 +124,17 @@ The following arguments are supported:
   operations.  
   Defaults to **false**. Changing this will create a new resource.
 
+* `power_action` - (Optional, String) Specifies the power action to be done for the desktop.
+  The valid values are **os-start**, **os-stop**, **reboot**, **os-hibernate**.
+
+  -> The `power_action` is a one-time action.
+     Currently , only Windows is supported for `os-hibernate`.
+  
+* `power_action_type` - (Optional, String) Specifies the power action mechanisms for the desktop.
+  The valid values are as follows:
+  + **SOFT**: Normal operation.
+  + **HARD**: Forced operation.
```

Comment:
When `power_action_type` is updated, the corresponding action is also executed. This behavior is not described in the document.

Reply:
fixed